middle_end/flambda2/reaper/rebuild.ml

```diff
@@ -47,6 +47,10 @@ type param_decision =
   | Delete
   | Unbox of Variable.t DS.unboxed_fields Field.Map.t
 
+(* CR sspies: Throughout this file, we create bound paramters and variables
+   without corresponding debugging uids. Does it make sense to properly
+   propagate debugging uids there? If so, where should they come from? *)
```

Comment:
As discussed (re. wiki or CR labelling)

Reply:
We decided just to grep for `Flambda_debug_uid.none` when we want to improve things